language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/CodeActionModuleId.java

```diff
@@ -67,6 +67,17 @@ public String orgName() {
         return orgName;
     }
 
+    /**
+     * Get the Package name of this module ID.
+     *
+     * @return Package name
+     */
+    @Override
+    public String packageName() {
+        // TODO: Implement this method.
+        throw new UnsupportedOperationException("Not yet implemented");
```

Comment:
@IMS94 Please note

Reply:
Shall we create an issue to track this as well?

Reply:
Created https://github.com/ballerina-platform/ballerina-lang/issues/31459 to track this